.github/pull_request_template.md

```diff
@@ -18,20 +18,18 @@ Please follow the Title Formats below when you open a new PR:
 <!-- Uncomment this line if some issues to close -->
 <!-- Close #<issue number> -->
 
-> Problem Summary:
-
 ### What's changed and how it works?
 
 <!-- Uncomment this line if this PR is associated with a proposal -->
 <!-- Proposal: [name](url) -->
 
-> What's Changed:
-
 ### Related changes
 
 - [ ] Need to update `chaos-mesh/website`
-- [ ] Need to update `Chaos Dashboard`
-- [ ] Need to **cheery-pick to release branches**
+- [ ] Need to update `Dashboard UI`
+- Need to **cheery-pick to release branches**
+  - [ ] release-2.0
```

Comment:
Does that mean we have to constantly update the newest releases here?

Reply:
Yep. But not too often, only modify it whenever we release the next.